model/monitorCache/implementation/DeliveryMonitoringData.php

```diff
@@ -119,6 +120,34 @@ public function setDeliveryExecution(DeliveryExecution $deliveryExecution)
         $this->deliveryExecution = $deliveryExecution;
     }
 
+    /**
+     * @return DeliveryExecutionInterface
```

Comment:
The PHPDoc of the object variable should be also updated to `DeliveryExecutionInterface`. What do you think?

Reply:
Fixed